### ISSUE TITLE
Question about 'track_to_predict' in dict 'ret' of AV2 dataset.

### ISSUE THREAD

**Issue by ChengkaiYang** [vita-epfl/UniTraj#32]
Dear @Alan-LanFeng 
        I want to know the defination of 'track_to_predict' of AV2 dataset.I found 'track_to_predict' is not equal to 'AV' agent in Argoverse2 dataset. Some SOTA methods' paper on leaderboard AV2 said that 'AV' agent for each scenario is high quality. Does this mean they used 'AV' objects in each scenario to train their models?
![image](https://github.com/user-attachments/assets/b085daa6-8a9d-499e-900e-4953e6d12218)
       1)Thus, Is it neccessary to add 'AV' agents into 'sample_list' which contains all agents we want to train? 
       2)Moreover, I've found when add 'objects_of_interest' agents into training, Autobot model got lower minFDE6 and minADE6 which might be the result of adding agents into training set.I found 'objects_of_interest' are all have a 'continuous_valid_length' of 110 but not all agents which have a 'continuous_valid_length' of 110 are in 'objects_of_interests'.In your previous reply, 'objects_of_interest' means 'focal agent'.Is it neccessary to add other agent with 110 continuous_valid_length into training?
![image](https://github.com/user-attachments/assets/315854e7-14d5-4c29-acb5-a8b4bef03e91)
       Since Unitraj has been the most popular motion forecasting framework now, it might be neccessary to make those questions clear in order to get same competitive result as AV2 leaderboard.
       Thank you for your patience!

**Comment by Alan-LanFeng** [comment]
Hi,

250k high quality trajectories == focal agents in AV2 == tracks_to_predict in UniTraj
Multi-agent trajectories == objects_of_interest in UniTraj.
Thanks for the suggestions, we'll make it clear in the doc.

It is not necessary to include AV traj in training samples, and it not a trivial task to select which trajectory should be used for training. So UniTraj only provides the ones used on the official benchmarks.

**Comment by ChengkaiYang** [comment]
> Hi,
> 
> 250k high quality trajectories == focal agents in AV2 == tracks_to_predict in UniTraj Multi-agent trajectories == objects_of_interest in UniTraj. Thanks for the suggestions, we'll make it clear in the doc.
> 
> It is not necessary to include AV traj in training samples, and it not a trivial task to select which trajectory should be used for training. So UniTraj only provides the ones used on the official benchmarks.

Thank you for your reply and I've understood it! I used to believe that some previous work didn't get competitive results because its sub-optimal model structure. But after I reviewed some SOTA work's code, I found SOTA methods almost utilized extra trajectories (AV trajectories and multi-agent trajectories) in single agent motion forecasting task training!Although they predicted only focal trajectory in testing but their SOTA results on single agent motion forecasting leaderboard might utilize extra trajectories in training dataset!
It's to be further discussed whether those extra trajectories should be utilized in single agent trajectory prediction. If someone use unitraj frame to compare their own model to baseline results on AV2 leaderboard, it might be a large gap which make people confused.But it's not the error of unitraj or model.
Moreover, how can we get left neighbour, right neighbour, entry_lanes and exit_lanes of map lanes as a matrix or  other batched representation.Some work use the adjecency matrix of lanes when doing M2A attention. Looking forward Unitraj to add these codes, thanks!
